fdbclient/NativeAPI.actor.cpp

```diff
@@ -835,10 +843,13 @@ Database Database::createDatabase( Reference<ClusterConnectionFile> connFile, in
 
 	DatabaseContext *db;
 	if(preallocatedDb) {
-		db = new (preallocatedDb) DatabaseContext(cluster, clientInfo, clientInfoMonitor, LiteralStringRef(""), TaskDefaultEndpoint, clientLocality, true, false, apiVersion);
+		db = new (preallocatedDb)
+		    DatabaseContext(cluster, clientInfo, clientInfoMonitor, LiteralStringRef(""), TaskDefaultEndpoint,
+		                    clientLocality, true, false, apiVersion, /* switchable */ true);
```

Comment:
I think this will include callers like tester class processes, backup/dr, fdbcli, etc. I think that all makes sense, but I just wanted to confirm that was the idea.

Reply:
Ping

Reply:
Yes, it does include those callers (I just took a look at all callers of this overload of Database::createDatabase and they all seem appropriate as switchable)

The idea was more to make it the default, and have exceptions for anything that might sensibly read a key without first getting a read version. I think storageserver is the only such exception.